### PR TITLE
Updated version library, enable or disable prereleases

### PR DESCRIPTION
**Pull request by rxbn**
**testing PENDING**

- use packaging.version instead of semver library
- fix typos
- add ability to enable or disable prereleases

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Helminator takes the following environment variables:
 |Variable|Description|Example|
 |:--------|:-----------|:-------|
 |`HELMINATOR_ROOT_DIR`|Directory to scan|`/path/to/playbook`|
+|`HELMINATOR_ENABLE_PRERELEASES`|Configure Helminator to process prereleases|`true` or `false`|
```
Comment (containeroo-gists) [inline README.md]: `HELMINATOR_ENABLE_PRERELEASES`|Configure Helminator to process prereleases|`true` or `false` (default: `false`) |

**Review by containeroo-gists — APPROVED** [review]
lgtm

**Review by gi8lino — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (rxbn) [reply README.md]: done

**Review by gi8lino — APPROVED** [review]
lgtm

**Pull request closed without merging**